### PR TITLE
Updated Java SE documentation to include most recent Java releases

### PR DESCRIPTION
**Pull request by NottyCode**

### CODE REVIEW

File: modules/ROOT/pages/java-se.adoc [hunk 1]
```diff
@@ -10,18 +10,65 @@
...
+|17
+|&check;
+|
+|https://adoptium.net/?variant=openjdk17&jvmVariant=hotspot[Eclipse Adoptium 17]
+|https://docs.oracle.com/en/java/javase/17/migrate/toc.htm[Java SE 17 migration guide]
+|===
+
+== Migration tools
+
+In addition to the Java migration guides there are also some tools available that can provide advice targeted to your applications.
```
Comment (cthigh) [inline modules/ROOT/pages/java-se.adoc]: comma after guides
In addition to the Java migration guides, there

File: modules/ROOT/pages/java-se.adoc [hunk 2]
```diff
@@ -10,18 +10,65 @@
...
+The https://www.ibm.com/support/pages/node/6250913[Migration Toolkit for Application Binaries] can scan a WAR or EAR file and provide advice on migrating to new Java SE versions.
```
Comment (cthigh) [inline modules/ROOT/pages/java-se.adoc]: The tool will also scan JAR files but you must use the `--analyze` parameter below.  Suggestion:

The https://www.ibm.com/support/pages/node/6250913[Migration Toolkit for Application Binaries] can scan WAR,  EAR, or JAR files and provide advice on migrating to new Java SE versions.

File: modules/ROOT/pages/java-se.adoc [hunk 3]
```diff
@@ -10,18 +10,65 @@
...
+Just https://www.ibm.com/support/pages/node/6250913[download] and install the tool and then run it using:
+
+```
+java -jar binaryAppScanner.jar myapp.war --sourceJava=ibm8 --targetJava=java11
```
Comment (cthigh) [inline modules/ROOT/pages/java-se.adoc]: java -jar binaryAppScanner.jar myapp.war --analyze --sourceJava=ibm8 --targetJava=java11

Comment (cthigh) [reply modules/ROOT/pages/java-se.adoc]: if you don't want to include JAR files, you can leave off --analyze.  By default, the binary scanner does an --all report that doesn't support JAR files.

**Comment by gjwatts** [comment]
We can add this link to the doc for more information on performance benefits of OpenJ9 over Hotspot -> https://www.eclipse.org/openj9/performance/

**Review by dmuelle — APPROVED** [review]

**Review by cthigh — COMMENTED** [review]
migration tools changes look good.

**Pull request merged**